geode-core/src/distributedTest/java/org/apache/geode/internal/statistics/StatisticsDistributedTest.java

```diff
@@ -490,6 +491,49 @@ public void testPubAndSubCustomStats() throws Exception {
         combinedUpdateEvents == combinedPuts);
   }
 
+  @Test
+  public void testExistenceOfVMStats() {
+    String regionName = "region_1";
+    VM vm = getHost(0).getVM(0);
+
+    vm.invoke(() -> {
+      Properties props = new Properties();
+      props.setProperty(STATISTIC_SAMPLING_ENABLED, "true");
+      props.setProperty(STATISTIC_SAMPLE_RATE, "1000");
+
+      InternalDistributedSystem system = getSystem(props);
+
+      // assert that sampler is working as expected
+      GemFireStatSampler sampler = system.getStatSampler();
+      assertTrue(sampler.isSamplingEnabled());
```

Comment:
We should make sure the new test is at least written in AssertJ. If the effort to convert the other tests is small please consider that.

Reply:
ah, defeated by copy-paste. gonna fix it all